src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchForm"
+      data-qa="big"
+      method="post"
+      action="#"
+    >
+      <div class="searchBar">
+        <label for="bigBar">
+          <img
+            class="icon bigIcon"
+            src="images/Search.svg"
+            alt="icon"
+          >
+        </label>
+        <input
+          class="bigSearchBar search"
```

Comment:
```suggestion
          class="bigSearch search"
```
you have `div` with a class `searchBar`, and using here `bigSearchBar` can be confusing

Reply:
Understood.